manage_proj_edit_page.php

```diff
@@ -75,6 +75,11 @@
 
 $f_project_id = gpc_get_int( 'project_id' );
 $f_show_global_users = gpc_get_bool( 'show_global_users' );
+$f_hide_obsolet = gpc_get_bool( 'hideobsolet' );
+
+# OFF = show obsolet versions, anything else = hide them
+$c_hide_obsolet = ( $f_hide_obsolet == OFF ) ? OFF : ON;
+$t_hide_obsolet_filter = '&amp;hideobsolet=' . $c_hide_obsolet;
```

Comment:
use `&` not `&amp;` otherwise there when you call print_link_button() below, the Edit button's URL will have double encoding and the flag will not be passed on correctly:
`http://path.to/mantis/manage_proj_edit_page.php?project_id=132&amp;hideobsolet=1`